docs/content/preview/benchmark/sysbench-ysql.md

```diff
@@ -22,22 +22,46 @@ sysbench is a popular tool for benchmarking databases like PostgreSQL and MySQL,
 
 To ensure the recommended hardware requirements are met and the database is correctly configured before benchmarking, review the [deployment checklist](../../deploy/checklist/).
 
-Install sysbench using the following steps:
-
-```sh
-$ cd $HOME
-$ git clone https://github.com/yugabyte/sysbench.git
-$ cd sysbench
-$ ./autogen.sh && ./configure --with-pgsql && make -j && sudo make install
-```
-
-This installs the sysbench utility in `/usr/local/bin`.
-
-Make sure you have the [YSQL shell](../../api/ysqlsh/) `ysqlsh` exported to the `PATH` variable.
-
-```sh
-$ export PATH=$PATH:/path/to/ysqlsh
-```
+### Install sysbench
+
+Install sysbench on a machine which satisfies the Prerequisites using one of 
+the following options:
+
+<ul class="nav nav-tabs nav-tabs-yb">
+    <li>
+    <a href="#github" class="nav-link active" id="github-tab" data-bs-toggle="tab" role="tab" aria-controls="github" aria-selected="true">
+      <i class="fab fa-github" aria-hidden="true"></i>
+      Source
+    </a>
+  </li>
+  <li>
+    <a href="#rhel" class="nav-link" id="rhel-tab" data-bs-toggle="tab" role="tab" aria-controls="rhel" aria-selected="true">
+      <i class="fa-brands fa-redhat" aria-hidden="true"></i>
+      RHEL
+    </a>
+  </li>
+  <li >
+    <a href="#macos" class="nav-link" id="macos-tab" data-bs-toggle="tab" role="tab" aria-controls="macos" aria-selected="true">
+      <i class="fa-brands fa-apple" aria-hidden="true"></i>
+      macOS
+    </a>
+  </li>
+
+</ul>
+
+<div class="tab-content">
+  <div id="github" class="tab-pane fade show active" role="tabpanel" aria-labelledby="github-tab">
+{{% readfile "./github.md" %}}
```

Comment:
For short sections like these is it not simpler to just put the content inline?

Reply:
Will that tab flow still work? In the doc, the suggestion was to make it look like this: https://docs.yugabyte.com/preview/yugabyte-voyager/install-yb-voyager/#install-yb-voyager .

Reply:
Updated the file.